IMDA-BTG/aiverify
# fix order of widgets for classification report

## Description

Fix order of widgets in classification report.

## Motivation and Context

See https://github.com/IMDA-BTG/aiverify/issues/122

## Type of Change













## How to Test

Install the updated stock report plugin and verify that the report widgets in the classification template is now in the right order.

## Checklist

Please check all the boxes that apply to this pull request using "x":


## Screenshots (if applicable)

[If the changes involve visual modifications, include screenshots or GIFs that demonstrate the changes.]

## Additional Notes

[Add any additional information or context that might be relevant to reviewers.]
